scripts/generateCliDoc.js

```diff
@@ -195,18 +190,23 @@ function generateDoc() {
 
 }
 
-function splitString(command, temp, description) {
-	command = temp.split(/[^\w.,]\s(?=[A-Z])/)[0].trim();
-	description = temp.split(/[^\w.,]\s(?=[A-Z])/)[1].trim();
-	return { command, description };
+function splitString(temp) {
+	let command, description, details;
+
+	const match = temp.trim().split("  ").filter((s) => s).map((s) => s.trim());
```

Comment:
Wouldn't it make more sense to first trim and then filter? Otherwise we may again end up with empty strings. Or is this expected in some cases?

Reply:
Don't you think it is a better approach to do the filter first (many items are reduced to an array with a maximum of 3 items) and then trim the starting and ending spaces that could be left?
Otherwise we may trim a lot empty string items what would be unnecessary.